sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -85,3 +85,23 @@
 --    arguments: none
 --    behavior: can take a long time.
 
+
+
+#IfMissingColumn list_options edit_options
+  ALTER TABLE `list_options` ADD `edit_options` TINYINT(1) NOT NULL DEFAULT '1';
+#Endif
```

Comment:
actually disregard above. not sure why this is showing as new code since it already exists in the codebase.